docs/src/global/componentApiTable.jsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/no-danger */
+
+// import './ComponentApiTable.scss';
```

Comment:
Should we remove this if it's not needed?  Are we still iterating a bit on the updated documentation styling and such?

Reply:
Removing.

Reply:
Removed in 9aa7069.  Resolving.